app/controllers/orders_controller.rb

```diff
@@ -1,16 +1,15 @@
 class OrdersController < ApplicationController
   def new
-    @order = Order.new
     cart = Cart.find_by(session_key: session.id.to_s)
```

Comment:
To ensure that you won't get nil-error, maybe it would be better to use `cart = Cart.find_by!`

Reply:
I change that code. If cart is nul or items is empty -> redirect user to catalog with flash message